src/biogeochem/CNPhenologyMod.F90

```diff
@@ -2114,9 +2304,17 @@ subroutine CropPhenology(num_pcropp, filter_pcropp                     , &
                ! AgroIBIS uses a complex formula for lai decline.
                ! Use CN's simple formula at least as a place holder (slevis)
 
-            else if (hui(p) >= huigrain(p)) then
+            else if (hui(p) >= huigrain(p) .and. cphase(p) >= cphase_leafemerge) then
```

Comment:
Previous versions of CLM allowed the reproductive (grain filling) phase to begin before the crop entered the vegetative (leaf expansion) phase. This version requires at least one timestep in the reproductive phase before moving on.